compiler/common-artifacts/src/TestDataGenerator.cpp

```diff
@@ -74,6 +76,19 @@ template <> void geneate_random_data<bool>(std::mt19937 &gen, void *data, uint32
   }
 }
 
+template <typename T>
+void geneate_random_range(void *data, uint32_t size, int32_t range_min, int32_t range_max)
+{
```

Comment:
(Optional) just for safety? :D

```suggestion
{
  assert(range_min <= range_max);
```

Reply:
Taking care of `range_min` and `range_max` should exist instead of `assert` in `random_range*` functions. It could do nothing or throwing an error or something.

Reply:
I think assert is good to have :)